dll/ntdll/ldr/ldrinit.c

```diff
@@ -1542,8 +1617,8 @@ LdrpInitializeProcess(IN PCONTEXT Context,
     /* Save the old Shim Data */
     OldShimData = Peb->pShimData;
 
-    /* Clear it */
-    Peb->pShimData = NULL;
+    /* ReactOS specific: do not clear it. (Windows starts doing the same in later versions) */
+    //Peb->pShimData = NULL;
```

Comment:
Why? Is it reused elsewhere afterwards?

Reply:
I guess it will be used later

Reply:
@learn-more : What about RosAppcompatGetCompatVersion?